src/functions-and-arrays.js

```diff
@@ -78,8 +160,19 @@ const wordsCount = [
   'matter'
 ];
 
-function howManyTimes() {}
-
+function howManyTimes(wordsArr, wordToFind) {
+  if (wordsArr.length === 0) {
+    return 0;
+  } else {
+    let count = 0;
+    wordsArr.forEach(function (word) {
+      if (word === wordToFind) {
+        count++;
+      }
+    })
+    return count;
+  }
+}
```

Comment:
Only a suggestion for alternative layout of the code -> to make it easier to read.

- First thing, you can rule out all the edge case.
Plus, if you are returning only one line -> you can use below syntax

```
if (wordsArr.length === 0) return 0;
```

- And after do remaining logic 
```
 let count = 0;
    wordsArr.forEach(function (word) {
      if (word === wordToFind) {
        count++;
      }
    })
    return count;
    
```

Again, just a suggestion, but you write as you prefer ;)